testdata/p4_16_samples/issue1670-bmv2.p4

```diff
@@ -0,0 +1,75 @@
+/*******************************************************************************
```

Comment:
This is a proprietary license header, not an Apache license header. So either this file should not be here or the header is wrong.